utils/constants.tsx

```diff
@@ -39,7 +39,9 @@ import monokaiCSS from '!!file-loader?name=files/code_themes/[hash].[ext]!highli
 import solarizedDarkCSS from '!!file-loader?name=files/code_themes/[hash].[ext]!highlight.js/styles/solarized-dark.css';
 
 // eslint-disable-line import/order
-import solarizedLightCSS from '!!file-loader?name=files/code_themes/[hash].[ext]!highlight.js/styles/solarized-light.css'; // eslint-disable-line import/order
+import solarizedLightCSS from '!!file-loader?name=files/code_themes/[hash].[ext]!highlight.js/styles/solarized-light.css';
+
+export type ValueOf<T> = T[keyof T];
```

Comment:
This generic type allows getting union of values from a const object in a single line: `ValueOf<typeof my_obj>`

Reply:
I think we're good with this. Thanks :)